libr/core/cmd_anal.c

```diff
@@ -4583,10 +4583,9 @@ static void _anal_calls(RCore *core, ut64 addr, ut64 addr_end) {
 	ut8 buf[4096];
 	ut8 block[4096];
 	bufi = 0;
-	if (addr_end - addr > 0xffffff) {
+	if (addr_end - addr > UT32_MAX) {
```

Comment:
not the same

Reply:
yup , i fixed it , in this binary maxsecW7.exe , as we have merged all sections
from - to range is more than oxffffff , so acc would never analyze that bin at all .